packages/contracts-bedrock/src/dispute/lib/LibUDT.sol

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.15;
 
+// nosemgrep: sol-style-no-bare-imports
```

Comment:
Why do we ignore this?

Reply:
it'll make the unused imports script fail for this file

Reply:
Interesting, why? That sounds like a bug

Reply:
the 2 imports LibPosition and Position are imported from LibUDT to other files but are not used by LibUDT itself so it's an unused import

Reply:
Ahhh... ok I think that's something worth fixing. I don't really like when stuff gets imported like this since it's confusing. I see that `Position` is being used inside of `LibUDT` so at least we can keep that specific import.

Reply:
fixed